client/my-sites/checklist/onboardingChecklist.js

```diff
@@ -5,81 +5,89 @@
  */
 import page from 'page';
 import { isDesktop } from 'lib/viewport';
+import i18n from 'i18n-calypso';
 
 /**
  * Internal dependencies
  */
 const tasks = {
 	avatar_uploaded: {
-		title: 'Upload your profile picture',
-		description:
-			'Who’s the person behind the site? Personalize your posts and comments with a custom profile picture.',
-		duration: '2 mins',
-		completedTitle: 'You uploaded a profile picture',
-		completedButtonText: 'Change',
+		title: i18n.translate( 'Upload your profile picture' ),
+		description: i18n.translate(
+			'Who’s the person behind the site? Personalize your posts and comments with a custom profile picture.'
+		),
+		duration: i18n.translate( '%d min', '%d mins', { count: 2, args: [ 2 ] } ),
```

Comment:
For a static string like `2 mins`, do we need to use the number format, or just `translate( "2 mins" )`?

Reply:
I think the idea is to 'translate `%d min` (and `%d mins`) once, re-use for all numbers'.